cpp/modmesh/view/R3DWidget.hpp

```diff
@@ -111,6 +111,10 @@ class R3DWidget
     Qt3DExtras::Qt3DWindow * view() { return m_view; }
     RScene * scene() { return m_scene; }
     Qt3DRender::QCamera * camera() { return m_view->camera(); }
+    void resetCamera(Qt3DRender::QCamera * camera,
+                     float positionX = 0.0f,
```

Comment:
I am not sure using default parameters here is good for maintenance.  If no one objects, we can leave it here and observe.  (My hunch feeling is that it's gonna bite us soon.  We'll see.)